File: viasoft.licensing.customerlicensing/Viasoft.Licensing.CustomerLicensing.Domain/DTOs/Environment/OrganizationUnitEnvironmentOutput.cs
using System;

namespace Viasoft.Licensing.CustomerLicensing.Domain.DTOs.Environment
{
    public class OrganizationUnitEnvironmentOutput
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsProduction { get; set; }
        public bool IsWeb { get; set; }
        public bool IsDesktop { get; set; }
        public bool IsMobile { get; set; }
        public string DatabaseName { get; set; }
        public Guid OrganizationUnitId { get; set; }
        public Guid OrganizationId { get; set; }
        public Guid TenantId { get; set; }
        public bool IsActive { get; set; }
        public string DesktopDatabaseVersion { get; set; }
    }
}

File: viasoft.licensing.licensingmanagement-frontend/apps/licensing-management/src/clients/licensing-management/model/organizationUnitEnvironmentOutput.ts
export interface OrganizationUnitEnvironmentOutput { 
    name?: string | null;
    description?: string | null;
    isProduction?: boolean;
    isMobile?: boolean;
    isWeb?: boolean;
    isDesktop?: boolean;
    databaseName?: string | null;
    organizationUnitId?: string;
    organizationId?: string;
    tenantId?: string;
    isActive?: boolean;
    id?: string;
}
